EmpleoDotNet/Services/Social/Twitter/TwitterService.cs

```diff
@@ -41,7 +41,7 @@ public TwitterService()
 
             var message = $"Se busca: {title} http://emplea.do/JobOpportunity/Detail/{UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)}";
 
-            await PostTweet(message);
+            await PostTweet(message).ConfigureAwait(false);
```

Comment:
Not really. Cada `ConfigureAwait` que defines está configurando el `await` keyword que espera el task hasta que se ejecute. 

Hay formas de no tener que escribirlo cada vez: 
http://stackoverflow.com/questions/27109285/is-there-a-more-readable-alternative-to-calling-configureawaitfalse-inside-an